Makefile.nt

```diff
@@ -180,9 +180,11 @@ endif
 	if test -n "$(WITH_DEBUGGER)"; then \
 	  $(MAKE) -C debugger install; \
 	fi
+ifeq "$(UNIX_OR_WIN32)" "win32"
 	if test -n "$(FLEXDLL_SUBMODULE_PRESENT)"; then \
```

Comment:
I'm not fond of having half of the test as gnu make conditional and the other one as a shell one. Could you make everything either gnu make conditionals or shell conditionals?